### PR TITLE
Improved top bar, menu and sign up.

### PR DESCRIPTION
**Pull request by aduermael**
- Top bar is less crowded now (correct icon for friends in on the 4th screenshot)
- Cubzh menu better organized also with quick social actions and settings on the right side. The central part still deserves some work, but it's a bit easier to read in the meantime.
- Sign up screen improvements: 
    - Combo boxes now better occupy the space
    - Combo boxes remember selection (auto scroll) when opening them
    - Checking username is faster
    - Added a few emojis
    - Sign Up button is bigger
    - Login and Help buttons are more discreet
    - Default placeholder color is easier to read (we couldn't do semi-transparent white before due to blending issue)
    

Screenshot 2023-12-09 at 20 29 22

Screenshot 2023-12-11 at 07 28 18

Screenshot 2023-12-11 at 07 28 27

Screenshot 2023-12-11 at 07 30 01

### CODE REVIEW

File: lua/modules/menu.lua [hunk 1]
```diff
@@ -42,6 +49,14 @@ modalWasShown = false
 alertWasShown = false
 cppMenuIsActive = false
 chatDisplayed = false -- when false, only a mini chat console is displayed in the top bar
+_DEBUG = false
+_DebugColor = function()
+	return Color(
+		150 + math.floor(math.random() * 105),
```
Comment (caillef) [inline lua/modules/menu.lua:55]: here we can replace by a simpler form `math.random(150,255)`

**Review by caillef — COMMENTED** [review]
One smal lcomment but the rest LGTM

File: lua/modules/menu.lua (hunk 1 above)
Comment (aduermael) [reply lua/modules/menu.lua]: Done!

**Review by caillef — APPROVED** [review]
LGTM!

**Pull request merged**